src/packages/emmett/src/testing/shoppingCart.feature

```diff
@@ -0,0 +1,17 @@
+Feature: ShoppingCart
```

Comment:
**QUESTION:** Out of curiosity, is this the way how you typically document testing scenarios, or is it also some longer shot behind it (e.g. DSL for testing)? Disclaimer, I really like that you documented it!

Reply:
Hey @oskardudycz! Well, this is typically how I do it when working on a product - I first write down the acceptance criteria of a task like this. However, I did it here to clarify the intent behind these "Shareable" tests. But I'm concerned about ensuring it's up-to-date with the implementation - maybe we should convert it to an internal executable DSL - thoughts?

Reply:
I was thinking initially about generating human-readable markdown or DSL from tests at some point, but we can do it either way. You mentioned that you did some DSL-based tests in Node. We can discuss that as a separate issue. I’m fine with having them as they are 🙂

Thank you for clarifications 👍